src/Data/Unit.elm

```diff
@@ -114,6 +116,12 @@ type Durability
     = Durability Float
 
 
+type alias HolisticDurability =
+    { physical : Durability
+    , nonPhysical : Durability
+    }
+
+
```

Comment:
I wonder if we shouldn't have two distinct types for `PhysicalDurability` and `NonPhysicalDurability`? and so the type `Durability` could be `{ physical : PhysicalDurability, nonPhysical : NonPhysicalDurability }`. The main benefit outside readability is to have the compiler check we don't mix these two values up.